### DISCUSSION TITLE
Fix for current methods

### DISCUSSION THREAD

**Discussion by 1-Mythical** [3kh0/ext-remover#458] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
It seems that trying all methods here isn't working. My school uses Blocksi and Lightspeed Filter Agent (Relay). The only method that kind of worked was the old Point Blank which really only made Blocksi disable for a split second. All of the other methods have some problem and don't work. The one I believe can work is LTBEEF inspect. I have bookmarklets that have a console, but not the ability to execute it. Also, if you have any solutions to getting bookmarklets to run on that manifest.json page, please respond. I am mainly just looking for a working method besides the insane ones like unenrolling and degrading.

**Comment by 3kh0** [comment, marked as answer]
there is not a way to run bookmarklets on ext pages, that is impossible (we think)

**Reply by 1-Mythical** [reply]
Any other way of getting inspect on that page?

**Reply by 1-Mythical** [reply]
Also, just a quick question. How is Ingot/LTBEEF broken anyways. Before, it would disable any extention, but now, it can't disable any admin extensions. I just want to know.

**Reply by Aka-but-nice** [reply]
They made it where the webstore can't disable admin extensions, now it can still disable regular, or sideload extensions, but not forced ones anymore.
